backend/de.metas.manufacturing/src/main/java/org/eevolution/productioncandidate/model/dao/PPOrderCandidateDAO.java

```diff
@@ -115,6 +128,22 @@ public ImmutableList<I_PP_OrderCandidate_PP_Order> getOrderAllocations(@NonNull
 				.collect(ImmutableList.toImmutableList());
 	}
 
+	@NonNull
+	public ImmutableList<I_PP_Order_Candidate> getCandidatesByOrderId(@NonNull final PPOrderId ppOrderId)
```

Comment:
rename it to getByOrderId.
This repo is about only candidates... so it's obvious we are talking about those.